assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -1360,6 +1361,13 @@ const baseSelectors = {
 		}
 
 		return Object.keys( modules ).reduce( ( acc, slug ) => {
+			if (
```

Comment:
I realise this is following the IB as specced, but it strikes me it might be marginally safer to move this block within the conditional block where `shareable` and `internal` have been checked. Probably not really needed - it seems very unlikely we'd remove shareability from Analytics, but I thought it's worth a mention seeing as the PR is going back to you anyway. Let me know what you think - I think either way is fine, really.

Reply:
Hi folks!

While this change absolutely makes sense, if UA is not connected in the Analytics module, the `analytics` module actually comes up as not `shareable` because of the `$this->is_connected()` condition here:

https://github.com/google/site-kit-wp/blob/216103df16fe39ae25f85573bcaa6cd93addbc13/includes/Core/Modules/Module.php#L771-L782

This is why, I thought placing it at the top-level would be ideal.

Let me know what you think.

Reply:
Thanks @nfmohit, as discussed elsewhere this makes sense and @jimmymadon has created a followup [PR](https://github.com/google/site-kit-wp/pull/6826) to address it.